feature_layers/create-mobile-geodatabase/src/main/java/com/esri/samples/create_mobile_geodatabase/CreateMobileGeodatabaseController.java

```diff
@@ -0,0 +1,336 @@
+import com.esri.arcgisruntime.geometry.Multipoint;
+import com.esri.arcgisruntime.geometry.Point;
+import com.esri.arcgisruntime.geometry.PointCollection;
+import com.esri.arcgisruntime.geometry.SpatialReferences;
+import com.esri.arcgisruntime.layers.FeatureLayer;
+import com.esri.arcgisruntime.mapping.view.Graphic;
+import com.esri.arcgisruntime.mapping.view.GraphicsOverlay;
+import com.esri.arcgisruntime.symbology.SimpleMarkerSymbol;
+import com.esri.arcgisruntime.data.QueryParameters;
+
+public class CreateMobileGeodatabaseController {
+
+  @FXML private MapView mapView;
+  @FXML private Label label;
+  @FXML private Button viewTableButton, createGeodatabaseButton, closeGeodatabaseButton;
+  @FXML private Stage tableStage;
+
+  private ArcGISMap map;
+  private GraphicsOverlay graphicsOverlay;
+  private Geodatabase geodatabase;
+  private Path geodatabasePath;
+  private GeodatabaseFeatureTable geodatabaseFeatureTable;
+  private List<Point> inputs;
+  private boolean isTableWindowOpen;
+
+  @FXML
+  private void initialize() {
+
+    try {
+      // authentication with an API key or named user is required to access base maps and other location services
+      String yourAPIKey = System.getProperty("apiKey");
+      ArcGISRuntimeEnvironment.setApiKey(yourAPIKey);
+
+      // create a map with the topographic imagery basemap style
+      map = new ArcGISMap(BasemapStyle.ARCGIS_TOPOGRAPHIC);
+
+      // set the map to the mapview
+      mapView.setMap(map);
+
+      // create a point located at Harper's Ferry, West Virginia to be used as the viewpoint for the map
+      var point = new Point(-77.7332, 39.3238, SpatialReferences.getWgs84());
+      mapView.setViewpointCenterAsync(point, 15000);
+
+      // create a graphics overlay to display the input points
+      graphicsOverlay = new GraphicsOverlay();
+      mapView.getGraphicsOverlays().add(graphicsOverlay);
+
+      // create a graphic to add the simple marker symbol in the graphics overlay
+      var simpleMarkerSymbol = new SimpleMarkerSymbol(SimpleMarkerSymbol.Style.CIRCLE, 0xFF000000, 10);
+      var graphic = new Graphic();
+      graphic.setSymbol(simpleMarkerSymbol);
+      graphicsOverlay.getGraphics().add(graphic);
+
+      // keep track of the points added by the user
+      inputs = new ArrayList<>();
+
+      // create a point from user input and add it to the geodatabase feature table
+      mapView.setOnMouseClicked(e -> {
+        if (createGeodatabaseButton.isDisabled()) {
+          if (e.isStillSincePress() && e.getButton() == MouseButton.PRIMARY) {
+            // create 2D point from pointer location
+            var point2D = new Point2D(e.getX(), e.getY());
+
+            // create a map point from 2D point
+            Point mapPoint = mapView.screenToLocation(point2D);
+
+            // the map point should be normalized to the central meridian when wrapping around a map, so its value stays within the coordinate system of the map view
```

Comment:
```suggestion
            // the map point should be normalized to the central meridian when wrapping around a map, so its value 
            // stays within the coordinate system of the map view
```
Add a new line at `stays` to keep the line length shorter